File: app/src/main/java/com/kanade/ushio/api/CalendarService.java
package com.kanade.ushio.api;

import com.kanade.ushio.entity.Calendar;

import java.util.List;

import io.reactivex.Observable;
import retrofit2.http.GET;

public interface CalendarService {
    @GET("/calendar")
    Observable<List<Calendar>> get();
}

File: app/src/main/java/com/kanade/ushio/api/CalendarService.kt
package com.kanade.ushio.api

import com.kanade.ushio.entity.Calendar
import io.reactivex.Flowable
import retrofit2.Call
import retrofit2.http.GET

interface CalendarService {
    @GET("/calendar")
    fun queryCalendar(): Flowable<List<Calendar>>

    @GET("/calendar")
    fun queryCalendarSync(): Call<List<Calendar>>
}